### PR TITLE
adding active flag to expense and destination attributes

### PR DESCRIPTION
**Pull request by labhvam5**

### CODE REVIEW

File: fyle_accounting_mappings/models.py [hunk 1]
```diff
@@ -189,16 +190,20 @@ def bulk_create_or_update_expense_attributes(
...
+                            attribute['active'] != primary_key_map[attribute['value']]['active']):
                     attributes_to_be_updated.append(
                         ExpenseAttribute(
                             id=primary_key_map[attribute['value']]['id'],
-                            detail=attribute['detail'],
+                            detail=attribute['detail'] if 'detail' in attribute else None,
+                            active=attribute['active'] if 'active' in attribute else None
                         )
                     )
         if attributes_to_be_created:
```
Comment (Sravanksk) [inline fyle_accounting_mappings/models.py:209]: in line 213 - you should update active field also 
`            ExpenseAttribute.objects.bulk_update(attributes_to_be_updated, fields=['detail', 'active'], batch_size=50)
`

Comment (labhvam5) [reply fyle_accounting_mappings/models.py:209]: Done

File: fyle_accounting_mappings/models.py [hunk 2]
```diff
@@ -189,23 +190,27 @@ def bulk_create_or_update_expense_attributes(
...
                         detail=attribute['detail'] if 'detail' in attribute else None,
-                        workspace_id=workspace_id
+                        workspace_id=workspace_id,
+                        active=attribute['active'] if 'active' in attribute else None
                     )
                 )
             else:
                 if update and 'detail' in attribute and \
-                        attribute['detail'] != primary_key_map[attribute['value']]['detail']:
+                        attribute['detail'] != primary_key_map[attribute['value']]['detail']or\
```
Comment (ashwin1111) [inline fyle_accounting_mappings/models.py:199]:
```
if update and (
  ('detail' in attribute and attribute['detail'] != primary_key_map[attribute['value']]['detail']) 
  or 
  ('active' in attribute and attribute['active'] != primary_key_map[attribute['value']]['active'])
)

File: fyle_accounting_mappings/models.py [hunk 3]
```diff
@@ -310,27 +316,31 @@ def bulk_create_or_update_destination_attributes(
...
+                        workspace_id=workspace_id,
+                        active=attribute['active'] if 'active' in attribute else None
                     )
                 )
             else:
                 if update:
                     if (attribute['value'] != primary_key_map[attribute['destination_id']]['value']) or \
                             ('detail' in attribute and
-                             attribute['detail'] != primary_key_map[attribute['destination_id']]['detail']):
+                             attribute['detail'] != primary_key_map[attribute['destination_id']]['detail'])or\
```
Comment (ashwin1111) [inline fyle_accounting_mappings/models.py:327]: same as above comment

**Review by Sravanksk — APPROVED** [review]

**Comment by Sravanksk** [comment]
@labhvam5 please resolve pylint errors and bump version before merging

**Pull request merged**